### PR TITLE
Add table gcp_sql_database_instance_metric_connections closes #266

### LINKED ISSUE #266: Add table gcp_sql_database_instance_connections
**References**
Add any related links that will help us understand the resource, including vendor documentation, related GitHub issues, and Go SDK documentation.

- Add table which will show information about database connection in 5 minute intervals

### PR DESCRIPTION
**Pull request by ParthaI**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
### Basic Info

```sql
select
  instance_id,
  timestamp,
  minimum,
  maximum,
  average,
  sample_count
from
  gcp_sql_database_instance_connections
```

```
+-----------------------+----------------------+---------+---------+---------+--------------+
| instance_id           | timestamp            | minimum | maximum | average | sample_count |
+-----------------------+----------------------+---------+---------+---------+--------------+
| parker-aaa:db-connect | 2021-06-28T10:46:00Z | 8       | 9       | 8.8     | 5            |
| parker-aaa:db-connect | 2021-06-28T10:51:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T10:56:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:01:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:06:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:11:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:16:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:21:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:26:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:31:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:36:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:41:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:46:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:51:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T11:56:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:01:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:06:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:11:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:16:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:21:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:26:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:31:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:36:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:41:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:46:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:51:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T12:56:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:01:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:06:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:11:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:16:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:21:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:26:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:31:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:36:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:41:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:46:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:51:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T13:56:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:01:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:06:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:11:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:16:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:21:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:26:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:31:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:36:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:41:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:46:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:51:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T14:56:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T15:01:00Z | 9       | 9       | 9       | 5            |
| parker-aaa:db-connect | 2021-06-28T15:06:00Z | 9       | 9       | 9       | 5            |

```

### Intervals averaging fewer than 9 connections

```sql
select
  instance_id,
  timestamp,
  round(minimum::numeric,2) as min_connection,
  round(maximum::numeric,2) as max_connection,
  round(average::numeric,2) as avg_connection,
  sample_count
from
  gcp_sql_database_instance_connections
where average < 9
```

```
+-----------------------+----------------------+----------------+----------------+----------------+--------------+
| instance_id           | timestamp            | min_connection | max_connection | avg_connection | sample_count |
+-----------------------+----------------------+----------------+----------------+----------------+--------------+
| parker-aaa:db-connect | 2021-06-28T10:43:00Z | 8.00           | 9.00           | 8.50           | 2            |
+-----------------------+----------------------+----------------+----------------+----------------+--------------+
```

</details>

### CODE REVIEW

File: docs/tables/gcp_sql_database_instance_metric_connections.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Table: gcp_sql_database_instance_metric_connections
```
Comment (rajmohanty17) [inline docs/tables/gcp_sql_database_instance_metric_connections.md:1]: should the table name be `gcp_sql_database_instance_connections`

File: gcp/table_gcp_sql_database_instance_metric_connections.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+	sqladmin "google.golang.org/api/sqladmin/v1beta4"
+)
+
+//// TABLE DEFINITION
+
+func tableGcpSQLDatabaseInstanceConnectionsMetric(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_sql_database_instance_metric_connections",
+		Description: "GCP SQL Database Instance Metrics Connections",
```
Comment (rajmohanty17) [inline gcp/table_gcp_sql_database_instance_metric_connections.go:17]: same here, should it be `gcp_sql_database_instance_connections`

**Review by rajmohanty17 — CHANGES_REQUESTED** [review]
Pls check once on the comments.

File: gcp/table_gcp_sql_database_instance_metric_connections.go (hunk 2 above)
Comment (Subhajit97) [reply gcp/table_gcp_sql_database_instance_metric_connections.go:17]: @rajmohanty17 We have followed same naming pattern defined in AWS

**Review by rajmohanty17 — APPROVED** [review]
LGTM

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**